src/main/java/de/zalando/ep/zalenium/registry/ZaleniumRegistry.java

```diff
@@ -55,6 +58,8 @@ public ZaleniumRegistry(Hub hub) {
         this.newSessionQueue = new NewSessionRequestQueue();
         proxies = new ProxySet((hub != null) ? hub.getConfiguration().throwOnCapabilityNotPresent : true);
         this.matcherThread.setUncaughtExceptionHandler(new UncaughtExceptionHandler());
+        Environment environment = new Environment();
+        maxRequestAge = environment.getIntEnvVariable("ZALENIUM_MAX_REQUEST_AGE", 5);
```

Comment:
I think seconds will be better here than minutes, because otherwise this is saying the shortest timeout you can have is 1 minute, but sometimes you might want to fail things earlier.

Reply:
Sounds good, just changed it